openmetadata-ui/src/main/resources/ui/playwright/utils/common.ts

```diff
@@ -147,3 +147,16 @@ export const clickOutside = async (page: Page) => {
   }); // with this action left menu bar is getting opened
   await page.mouse.move(1280, 0); // moving out side left menu bar to avoid random failure due to left menu bar
 };
+
+export const visitUserProfilePage = async (page: Page) => {
+  await page.getByTestId('dropdown-profile').click();
+  await page.waitForSelector('.profile-dropdown', {
+    state: 'visible',
+  });
+  const userResponse = page.waitForResponse(
+    '/api/v1/users/name/*?fields=profile%2Croles%2Cteams%2Cpersonas%2CdefaultPersona%2Cdomain&include=all'
```

Comment:
```suggestion
    '/api/v1/users/name/*?fields=*&include=all'
```

if fields change in the future it can cause unnecessary failure of test